core/dbt/contracts/graph/manifest.py

```diff
@@ -1528,7 +1528,8 @@ def get_manifest_schema_version(dct: dict) -> int:
     schema_version = dct.get("metadata", {}).get("dbt_schema_version", None)
     if not schema_version:
         raise ValueError("Manifest doesn't have schema version")
-    return int(schema_version.split(".")[-2][-1])
+
+    return int(schema_version.split("/")[-1].split(".")[0][1:])
```

Comment:
You've explained to me how this works, and it looks good. We should probably add a comment here documenting the format / an example of the string we're expecting. That will make it more obvious why this specific sequence of parsing actually does what we want it to do 🙂